server/plugins/connect-plugin-api/src/main/java/com/oceanbase/odc/plugin/connect/api/ConnectionExtensionPoint.java

```diff
@@ -33,7 +35,14 @@
  */
 public interface ConnectionExtensionPoint extends ExtensionPoint {
 
-    String generateJdbcUrl(String host, Integer port, String defaultSchema, Map<String, String> jdbcParameters);
+    /**
+     * @param properties Properties required by jdbcURL, such as HOST, PORT and DEFAULT_SCHEMA, see
+     *        {@link ConnectionConstants}
+     * @param jdbcParameters jdbc parameters.
+     *
+     * @return jdbcURL
+     */
+    String generateJdbcUrl(Properties properties, Map<String, String> jdbcParameters);
```

Comment:
why should we use `Properties` to input host and port? Is this stipulated in the jdbc protocol?